motoman_driver/include/motoman_driver/industrial_robot_client/joint_trajectory_streamer.h

```diff
@@ -129,11 +132,13 @@ class JointTrajectoryStreamer : public JointTrajectoryInterface
 
   boost::thread* streaming_thread_;
   boost::mutex mutex_;
-  int current_point_;
+  int current_point_, streaming_sequence;

```

Comment:
is this missing a trailing underscore?

Additional suggestion: what is this a sequence number of / for?